lib/validator/v3/validator.go

```diff
@@ -72,6 +72,7 @@ var (
 	overlapsV6LinkLocal   = "IP pool range overlaps with IPv6 Link Local range fe80::/10"
 	protocolPortsMsg      = "rules that specify ports must set protocol to TCP or UDP"
 	protocolIcmpMsg       = "rules that specify ICMP fields must set protocol to ICMP"
+	protocolAndHTTPMsg    = "rules that specify HTTP fields must set protocol to TCP"
```

Comment:
the wording might need some work…

Reply:
I think this looks alright to me - it matches the format of the other strings.

Maybe "can only set protocol to TCP" since "must" is a little bit strong?

or 

"rules that specify HTTP fields must set protocol to TCP or empty" or something like that?

Reply:
good good… I like the 2nd one better, I think.

Reply:
Updated.